internal/benchmarks/src/crypto.bench.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import type { Keystore } from '@fuel-ts/crypto';
+import { bufferFromString, pbkdf2, computeHmac, encrypt, decrypt } from '@fuel-ts/crypto';
```

Comment:
```suggestion
import type { Keystore } from 'fuels;
import { bufferFromString, pbkdf2, computeHmac, encrypt, decrypt } from 'fuels';
```

Pretty sure all these can come from `fuels`

Reply:
a0597b9d7